src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -106,15 +106,15 @@ public static void cameraPos(EntityViewRenderEvent.CameraSetup event) {
     @OnlyIn(Dist.CLIENT)
     @SubscribeEvent
     public static void bobViewer(RenderViewEvent event) {
-        Entity ridding = Minecraft.getInstance().player.getVehicle();
         Minecraft mc = Minecraft.getInstance();
-        CameraType cameraType = mc.options.getCameraType();
+        Entity ridding = mc.player.getVehicle();
 
         if (Methods.isRocket(ridding)) {
+            CameraType cameraType = mc.options.getCameraType();
             if (cameraType.equals(CameraType.THIRD_PERSON_FRONT) || cameraType.equals(CameraType.THIRD_PERSON_BACK)) {
                 event.setCanceled(true);
 
-                if (ridding instanceof IRocketEntity && ridding.getEntityData().get(IRocketEntity.ROCKET_START)) {
+                if (ridding.getEntityData().get(IRocketEntity.ROCKET_START)) {
```

Comment:
No need to check if riding a rocket, already checked before getting to this stage (line ~112 - `if (Methods.isRocket(ridding))`)